pkgs/development/tools/kdoctor/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  githubUser = "Kotlin";
```

Comment:
There is no need for parameterize everything. Use the value directly into the rest of the expression.